components/Wallet.tsx

```diff
@@ -24,12 +22,6 @@ import {
   useId
 } from "@floating-ui/react";
```

Comment:
aside: do we need this? not sure how this snuck in

Reply:
Yeah this is being used currently for the wallet tooltip/popup. Its a bit clunky and we should most likely replace it with the one you created in delta.